giskard/scanner/scanlogger.py

```diff
@@ -0,0 +1,56 @@
+import enum
+from typing import Optional
+
+from giskard.utils.xprint import (
+    Template,
+    xprint,
+)
+
+class ScanLoggerLevel(enum.Enum):
+    DEBUG = 0
+    INFO = 5
+    CRITICAL = 10
```

Comment:
I think you got these backward.

for instance `if level < ScanLoggerLevel.DEBUG.value` we want to show everything: DEBUG + INFO + CRITICAL for `if level < ScanLoggerLevel.INFO.value` we want to show only:  INFO + CRITICAL 
`if level < ScanLoggerLevel.CRITICAL.value` we want to show only:  CRITICAL 

see also: https://docs.python.org/3/library/logging.html#logging-levels

Reply:
Let's say that we are currently in `debug` mode, meaning that `self.level.value = 0`.
Then when we call `logger.info`, the condition `self.level.value <= ScanLoggerLevel.INFO.value` becomes `0 <= 5`, which is true so INFO logs are shown. Same for `critical` mode.
Conversely, if we choose `critical` mode, we get `self.level.value = 10` so no conditions are satisfied except for `logger.critical`.
I think it's working as intended.